lib/features/chat/domain/helix_get_channel_emotes.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+import 'helix_images.dart';
+
+part 'helix_get_channel_emotes.freezed.dart';
+part 'helix_get_channel_emotes.g.dart';
+
+@freezed
+class HelixGetChannelEmotesResponse with _$HelixGetChannelEmotesResponse {
+  const factory HelixGetChannelEmotesResponse({
+    required int total,
+    required List<HelixChannelEmote> data,
+    required String template,
+  }) = _HelixGetChannelEmotesResponse;
+  factory HelixGetChannelEmotesResponse.fromJson(Map<String, dynamic> json) =>
+      _$HelixGetChannelEmotesResponseFromJson(json);
+}
+
+@freezed
+class HelixChannelEmote with _$HelixChannelEmote {
+  @JsonSerializable(fieldRename: FieldRename.snake)
+  const factory HelixChannelEmote({
+    required String id,
+    required String name,
+    required HelixImages images,
+    required List<String> format,
+    required List<String> scale,
+    required List<String> themeMode,
+    required String emoteType,
```

Comment:
Make `emoteType` and `emoteSetId nullable. Add additional nullable fields `String? tier` and `String? owner_id`.